velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -223,6 +223,12 @@ class DateTimeFunctionsTest : public functions::test::FunctionBaseTest {
     }
     return signatureStrings;
   }
+
+  Date parseDate(const StringView& dateStr) {
```

Comment:
I believe there are existing tests that use this as a lambda. We can delete these lambdas now, right?